client/pages/_app.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint react/prop-types: 0 */
+import React from "react";
+import { FetchContextProvider } from "../context/FetchContext";
+import { PaginatorContextProvider } from "../context/PaginatorContext";
+
+import "../styles/normalize.css";
+import "../styles/application.css";
+import "../styles/client.css";
+import "../styles/justgrid.css";
+import "@fortawesome/fontawesome-free/css/all.min.css";
+
+function MyApp({ Component, pageProps }) {
+    return (
+        <PaginatorContextProvider>
```

Comment:
Does it make sense to put the PaginatorContextProvider around the whole application, or would it be better to put this only around the Paginator itself? (If this keeps things simpler, I don't mind it... but it seems potentially over-broad).

Reply:
Made change

Reply:
Ahh, I see, sorry for the false alarm. :-)